pages/common/bower.md

```diff
@@ -0,0 +1,32 @@
+# bower
+
+> A package manager optimized for the front-end web development.
```

Comment:
I'll go with "front-end development" as it's the key word with bower. Web development includes the backend which bower doesn't focus on.